changelog/64372.fixed.md

```diff
@@ -0,0 +1 @@
+Replace libnacl with PyNaCl
```

Comment:
This seems more like a `changed` changelog then fixing a bug. Can you update the file extension name to `changed.md`

Reply:
But could argue bug since libnacl not working.

Reply:
done, moved to changed